app/src/main/java/com/spielpark/steve/bernieapp/model/news/NewsManager.java

```diff
@@ -0,0 +1,54 @@
+package com.spielpark.steve.bernieapp.model.news;
+
+import android.util.Pair;
+
+import com.spielpark.steve.bernieapp.model.ApiManager;
+import com.spielpark.steve.bernieapp.model.BernieApi;
+
+import java.util.List;
+
+import rx.Observable;
+import rx.functions.Func2;
+import rx.schedulers.Schedulers;
+
+public class NewsManager {
+
+    private static NewsManager instance;
+    private final BernieApi api;
+
+    public NewsManager() {
+        api = ApiManager.get().api;
+
+    }
+
+    public static NewsManager get() {
+        if (instance == null) {
+            instance = new NewsManager();
+        }
+        return instance;
+    }
+
+    public Observable<Pair<NewsArticle, List<NewsArticle>>> getNews() {
```

Comment:
This looks much better, maybe some javadoc about why it's returning what it is sending?